Revert "Milestone 1.2 and 1.3: SVG generation for new entities."

U8NWXD opened the pull request:
Reverts oppia/oppia#9678 due to failures on develop:
* https://app.circleci.com/pipelines/github/oppia/oppia/11616/workflows/a05183d7-22fe-4e5c-8373-ee72b209b67f/jobs/66737
* https://app.circleci.com/pipelines/github/oppia/oppia/11607/workflows/1d6bbe56-6629-49b4-a082-2300d6bd44a2/jobs/66694

U8NWXD committed:
Revert "Milestone 1.2 and 1.3: SVG generation for new entities. (#9678)"

This reverts commit ce99bb04ba641c13d211a758f691b6ee90309f8e.

seanlip commented:
Per conversation with @U8NWXD we decided to wait for the tests to pass (except flaky e2e tests) before merging this.

U8NWXD committed:
Add waitFor Import Back to forms.js (#9796)

We used a waitFor in the chromedriver upgrade in #9267, so we need to
keep the waitFor import that was introduced by #9678.
